java-checks-aws/src/main/resources/org/sonar/java/checks/security/S6437-methods.json

```diff
@@ -7438,7 +7438,7 @@
   {"cls":"redis.clients.jedis.JedisSentinelPool","name":"JedisSentinelPool","args":["java.lang.String","java.util.Set","*","int","java.lang.String","int","java.lang.String"],"indices":[4]},
   {"cls":"redis.clients.jedis.JedisSentinelPool","name":"JedisSentinelPool","args":["java.lang.String","java.util.Set","*","int","java.lang.String","java.lang.String","int"],"indices":[5]},
   {"cls":"redis.clients.jedis.JedisSentinelPool","name":"JedisSentinelPool","args":["java.lang.String","java.util.Set","*","int","java.lang.String","java.lang.String","int","java.lang.String"],"indices":[5]},
-  {"cls":"redis.clients.jedis.JedisSentinelPool","name":"JedisSentinelPool","args":["java.lang.String","java.util.Set","*","java.lang.String"],"indices":[3]},
+  {"cls":"redis.clients.jedis.JedisSentinelPool","name":"JedisSentinelPool","args":["java.lang.String","java.util.Set","org.apache.commons.pool2.impl.GenericObjectPoolConfig","java.lang.String"],"indices":[3]},
```

Comment:
I think the entry with `*` that was removed here was covering both line 7441 and line 7443, maybe instead of replacing this entry we could remove line 7443 as it's redundant? 
I'm not sure if the intent of these entries is to prevent or to fix, the `*` approach would cover possible future new signatures, while being more specific with the naming allows to intervene only when needed.
I think we might want an opinion from  Hendrik here, as he's the main contributor to this list